nnstreamer.ini.in

```diff
@@ -8,6 +8,10 @@ enable_symlink=@ENABLE_SYMBOLIC_LINK@
 filters=@SUBPLUGIN_INSTALL_PREFIX@/filters/
 customfilters=@SUBPLUGIN_INSTALL_PREFIX@/customfilters/
 
+# Set framework priority about model file extension when automatically selecting framework for tensor filter.
+framework_priority_pb_tf=@FRAMEWORK_PRIORITY_PB@
+framework_priority_tflite_edgetpu=@FRAMEWORK_PRIORITY_TFLITE@
```

Comment:
It will be cheanged...

framework_priority_pb=@FRAMEWORK_PRIORITY_PB@
framework_priority_tflite=@FRAMEWORK_PRIORITY_TFLITE@